### DISCUSSION TITLE
留言板 | Super Blog

### DISCUSSION THREAD

**Discussion by giscus** [SuperPung/Blog#13] [General]
# 留言板 | Super Blog

生活 / 读书 / 新知

https://superpung.com/forum/

**Comment by superpung** [comment]
欢迎来到我的 Blog 😄

**Comment by M1chaelWang** [comment]
学长您好！最开始听说superpung这个id还是通过同学分享的课程分享计划，至今应该也有一年多的时间了。最近面临毕业，闲暇之余打开github翻看，找到了这个博客。作为一名tju大四软工学生，十分感谢您的付出。衷心祝您一切顺利！
